### DISCUSSION TITLE
I need some eyes on a PyPyODBC integration branch

### DISCUSSION THREAD

**Discussion by jvictore** [sqlalchemy/sqlalchemy#10699] [Usage Questions]
Hello everyone,

I currently have code that uses SQLAlchemy integrated with PyODBC, which I needed to switch to PyPyODBC. I noticed that SQLAlchemy currently does not have this integration with PyPyODBC (please correct me if I'm wrong).

As the intention of [PyPyODBC is to be easily interchangeable with PyOBC](https://github.com/pypyodbc/pypyodbc), while browsing the internet I found [this discussion](https://code.google.com/archive/p/pypyodbc/wikis/Enable_SQLAlchemy_on_PyPy.wiki) where it is said that to make this adaptation it is only necessary to change the names in the variables and functions. So, I forked this Repo and made the changes, it seems that my program continues to work normally.

Since I don't have experience with both libraries nor with OpenSource codes, I'd like to know how I can create a branch here and post my changes, so people can take a look and share their thoughts and concerns.

Thanks for your time!

**Comment by zzzeek** [comment]
assuming pypyodbc is **completely** cross-compatible with pyodbc, you can use pypyodbc right now without changing SQLAlchemy code [using module](https://docs.sqlalchemy.org/en/20/core/engines.html#sqlalchemy.create_engine.params.module):

```py
import pypyodbc
engine = create_engine("mssql+pyodbc://user:pass@dsn",  module=pypyodbc)

```

no forking / changing should be necessary
